.github/workflows/test.yml

```diff
@@ -0,0 +1,47 @@
+name: Tests
+on:
+  push:
+    branches: [ '*' ]
+  pull_request:
+    branches: [ main ]
+  schedule:
+    # Run automatically at 8AM PST Monday-Friday
+    - cron: '0 15 * * 1-5'
+  workflow_dispatch:
+
+jobs:
+  tests:
+    name: Run Tests
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    steps:
+      - name: Checkout sendgrid-csharp
+        uses: actions/checkout@v2
+
+      - name: Setup .NET Core SDK
+        uses: actions/setup-dotnet@v1.8.2
+        with:
+          dotnet-version: '3.1.x'
+
+      - run: dotnet build -c Release
+      - name: Build & Test
+        run: make test-docker release
+      - run: bash <(curl -s https://codecov.io/bash)
```

Comment:
My bad. Looks like we do need to set the fetch depth to 0
![image](https://user-images.githubusercontent.com/6811347/142698147-dc727032-f0de-4ce4-af2d-e76cd17bb8ac.png)

Reply:
Where are you seeing this error? Can't find it

Reply:
Latest run (22) under the "Run bash" step
![image](https://user-images.githubusercontent.com/6811347/142700020-919d77b0-0d8b-496d-a839-10942f81b8fe.png)

Reply:
aha